powerbi-docs/developer/visuals/capabilities.md

```diff
@@ -28,6 +29,40 @@ The root objects of a visual's capabilities are `dataRoles`, `dataViewMappings`,
 
 ```
 
+## Define the special permissions that your visual requires: privileges
+
+Privileges are the special operations required by your visual. It takes an array of `Privilege` objects, which defines all privilege properties:
```

Comment:
we should provide a table of currently supported privileges: external network calls, localStorage, FileDownload

Reply:
Added separate paragraphs for both privileges. I'm not sure this is the right place for specifying admin tenant switch